importer_config.template.py

```diff
@@ -1,129 +1,133 @@
-from os import path
 
```

Comment:
reformatted config file to be based on 2D array with commented sections and one class to fetch keys

Reply:
This works; I think one of the motivations to split out the .jsons was so that we had config in several files and not one big one? in any case, this is fine, we can split it if it starts feeling unweidly.